src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js

```diff
@@ -1,7 +1,7 @@
 import path from 'path';
 
-Cypress.Commands.add('checkSearch', () => {
-  cy.axeCheck();
+Cypress.Commands.add('checkSearch', ({ _13647Exception = false }) => {
+  cy.axeCheck('main', { _13647Exception });
```

Comment:
What does "13647Exception" mean?

Reply:
It means that the app is being given an exception for the stricter axe checks as part of ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/13647.

This is only temporary, and once this PR is merged I will set up separate tickets/PRs for each individual app teams to fix whatever is causing the axe checks to fail.

Reply:
To be clear, the name is intentionally wonky; it's not meant to be used by application teams writing new tests.

Reply:
I see. Well even so, maybe `temporaryException` would be more self-documenting?

Reply:
Will it be up to the facilities team to make the tests pass without the exception?

Reply:
> Will it be up to the facilities team to make the tests pass without the exception?

Yes. Once I merge this I will open a ticket, and even a PR for a branch where I remove the exception so that the failures are visible.